docs/03-tutorials/00-serverless/svls-11-set-asynchronous-connection-of-functions.md

```diff
@@ -59,23 +59,29 @@ This tutorial shows only one possible use case. There are many more use cases on
 >**NOTE:** In this example, there's no sanitization logic. The `sanitize` Function is just a placeholder.
 
    ```js
-   const { v4: uuidv4 } = require('uuid');
-   require('axios');
    module.exports = {
-       main: function (event, context) {
-           let sanitsedData = sanitise(event.data)
-           var eventOut=event.buildResponseCloudEvent(uuidv4(), "sap.kyma.custom.acme.payload.sanitised.v1", sanitisedData);
-           eventOut.source="kyma"
-           eventOut.specversion="1.0"
-           event.publishCloudEvent(eventOut);
-           console.log(`Payload pushed to sap.kyma.custom.acme.payload.sanitised.v1`,eventOut)
-           return eventOut;
-       }
+      main: async function (event, context) {
+         let sanitisedData = sanitise(event.data)
+
+         const eventType = "sap.kyma.custom.acme.payload.sanitised.v1";
+         const eventSource = "kyma";
+         
+         return await event.emitCloudEvent(eventType, eventSource, sanitisedData)
+               .then(resp => {
+                  if(resp.status!==204){
```

Comment:
Shouldn't we check the status code in `emitCloudEvent`. 
I see here leaking internals.

Reply:
you're absolutely right. this code is not needed. As we send  with axios the 4xx or 5xx responses are handled by the error handling. Removed that sample code